great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -419,3 +421,172 @@ def _build_numeric_rule() -> Rule:
         )
 
         return rule
+
+    @staticmethod
+    def _build_datetime_rule() -> Rule:
+        """
+        This method builds "Rule" object focused on emitting "ExpectationConfiguration" objects for datetime columns.
+        """
+
+        # Step-1: Instantiate "ColumnDomainBuilder" for selecting proper datetime columns (not "datetime-looking" text).
+
+        datetime_column_type_domain_builder: ColumnDomainBuilder = ColumnDomainBuilder(
+            include_column_names=None,
+            exclude_column_names=None,
+            include_column_name_suffixes=None,
+            exclude_column_name_suffixes=None,
+            semantic_type_filter_module_name=None,
+            semantic_type_filter_class_name=None,
+            include_semantic_types=[
+                SemanticDomainTypes.DATETIME,
+            ],
+            exclude_semantic_types=[
+                SemanticDomainTypes.TEXT,
```

Comment:
Would these "text datetime" columns be included if you didn't exclude them here? Why do we want to exclude them?

Reply:
@NathanFarmer We exclude them because this particular "DateTime" Rule handles bona-fide `datetime` fields, which are processed as numerical.  If a `TEXT` field is encountered, numerical operations, such as 'min', 'max', etc. will not be possible.  Moreover, the `datetime` looking `TEXT` rules have been deferred from the requirements, because the matching to formatted `DATETIME` strings is slow (when we go through all possibilities in the `ParameterBuilder`).  Thanks.